.goreleaser.yaml

```diff
@@ -11,10 +11,6 @@ builds:
   goarch:
   - amd64
   - arm64
-  ignore:
```

Comment:
I don't see why we shouldn't. This topic came up when testing the CLI in arm64 Docker containers.